core/systemcontracts/upgrade.go

```diff
@@ -738,7 +738,7 @@ func init() {
 	}
 }
 
-func UpgradeBuildInSystemContract(config *params.ChainConfig, blockNumber *big.Int, lastBlockTime uint64, blockTime uint64, statedb *state.StateDB) {
+func UpgradeBuildInSystemContract(config *params.ChainConfig, blockNumber *big.Int, lastBlockTime uint64, blockTime uint64, statedb vm.StateDB) {
 	if config == nil || blockNumber == nil || statedb == nil {
```

Comment:
would break this code "statedb == nil",

Reply:
@easyfold pls add a unit test for the senario to pass nil interface